src/client/components/sections/HeaderMovieSingle.jsx

```diff
@@ -50,7 +53,7 @@ class HeaderMovieSinglePage extends React.Component {
             );
         }
         const posterURL = movie.poster_path ? `https://image.tmdb.org/t/p/w500/${movie.poster_path}` : null;
-        const posters = movie.images || movie.images.posters ? movie.images.posters : null;
+        // const posters = movie.images || movie.images.posters ? movie.images.posters : null;
         const videos = movie.videos || movie.videos.results ? movie.videos.results : null;
```

Comment:
@vitkuz if i understand it right, it should be `movie.videos && movie.videos.resutls`. Right?

Reply:
Done